### PR TITLE
B2B configurable product tests

### PR DESCRIPTION
**Pull request by Nataliia-Popkova**
## PR Description

B2b configurable product tests

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: tests/ui/mp_b2b/catalog/catalog.robot [hunk 1]
```diff
@@ -608,58 +605,34 @@ Configurable_Product_RfQ_Order_Management
...
-    Zed: grand total for the order equals:    ${lastPlacedOrder}    €2,636.11
-    Zed: go to order page:    ${lastPlacedOrder}
-    Zed: product configuration should be equal:
-    ...    || shipment | position | sku                                  | date       | date_time ||
-    ...    || 1        | 1        | ${configurable_product_concrete_sku} | 12.12.2030 | Evening   ||
-    Zed: product configuration should be equal:
-    ...    || shipment | position | sku                                  | date       | date_time ||
-    ...    || 1        | 2        | ${configurable_product_concrete_sku} | 01.01.2050 | Morning   ||
-    Zed: trigger all matching states inside xxx order:    ${lastPlacedOrder}    Pay   
+    # Zed: grand total for the order equals:    ${lastPlacedOrder}    €771.90
```
Comment (AntonShopin) [inline tests/ui/mp_b2b/catalog/catalog.robot:608]: why it's here if commented?

File: tests/ui/b2b/catalog/catalog.robot [hunk 2]
```diff
@@ -550,86 +550,85 @@ Product_Availability_Calculation
...
+    ...    || 140        | 240        ||
+    Yves: save product configuration
+    Yves: shopping cart contains product with unit price:    ${configurable_product_concrete_sku}    ${configurable_product_name}    €2,206.54
+    Yves: delete all shopping carts
+    Yves: create new 'Shopping Cart' with name:    configProduct+${random}
+    Yves: go to PDP of the product with sku:    ${configurable_product_abstract_sku}
     Yves: add product to the shopping list:    configProduct+${random}
     Yves: go to 'Shopping Lists' page
     Yves: view shopping list with name:    configProduct+${random}
+    ## bug: https://spryker.atlassian.net/browse/CC-33647
```
Comment (AntonShopin) [inline tests/ui/b2b/catalog/catalog.robot:601]: ?

File: tests/ui/mp_b2b/catalog/catalog.robot [hunk 3]
```diff
@@ -490,90 +490,87 @@ Offer_Availability_Calculation
...
+    # ...    || 420        | 240        ||
+    # Yves: save product configuration
+    # Yves: configuration should be equal:
+    # ...    || option one | option two ||
+    # ...    || 6 shelves  | 2 lockers  ||
+    # Yves: add all available products from list to cart
+    # Yves: configuration should be equal:
+    # ...    || option one | option two ||
+    # ...    || 6 shelves  | 2 lockers  ||
+    # Yves: shopping cart contains product with unit price:    ${configurable_product_concrete_sku}    ${configurable_product_name}    €2,486.54
```
Comment (AntonShopin) [inline tests/ui/mp_b2b/catalog/catalog.robot:555]: if there is a bug that causes the test to fail - skip the whole test with tag and not just comment part of it

File: tests/ui/b2b/catalog/catalog.robot (hunk 2 above)
Comment (Nataliia-Popkova) [reply tests/ui/b2b/catalog/catalog.robot:601]: moved bug number to documentation section

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**